models.py

```diff
@@ -25,9 +25,11 @@ def get_todo(db: Session, item_id: int):
     return db.query(ToDo).filter(ToDo.id == item_id).first()
 
 
-def update_todo(db: Session, item_id: int, content: str):
+def update_todo(db: Session, item_id: int, content: str, notes: str = None):
     todo = get_todo(db, item_id)
     todo.content = content
```

Comment:
The `update_todo` function has been modified to include `notes`. Make sure to update any code that calls this function.